### PR TITLE
some updates to how docs look

### LINKED ISSUE #5425: Update CONTRIBUTING.md to point to actual contributing guide in docs
## Description of your problem
~~I'm trying to contribute docs but I'm not sure how to compile them. Contributing guide doesn't tell me how to go about this, just how to push changes, and mostly focuses on code changes~~

https://github.com/pymc-devs/pymc/blob/main/CONTRIBUTING.md

See @OriolAbril's comment below

### PR DESCRIPTION
**Pull request by OriolAbril**
Some updates to how docs look, and addressing some of the todos in the learning page, arguably the most important page in the docs. Closes #5425 

The preview is available at: https://pymc--5684.org.readthedocs.build/en/5684/index.html

@martinacantaro I tried to continue your GSoD work, I hope I haven't botched it too much

@drbenvincent and @reshamas I think this will address some of your comments

### CODE REVIEW

File: docs/source/learning.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 (learning)=
 # Learning
 
+## Installation
+Go to the {ref}`installation guide <installation>` to set up PyMC in your
```
Comment (canyon289) [inline docs/source/learning.md]:
```suggestion
Refer to the {ref}`installation guide <installation>` to set up PyMC in your
```

File: docs/source/learning.md [hunk 2]
```diff
@@ -52,29 +65,52 @@ PyMC's own {doc}`glossary` defines many core terms and provides useful reference
...
-:::
+#### Bayesian Modeling and Computation in Python
+
+By Osvaldo Martin, Ravin Kumar and Junpeng Lao
+
+`<add description here>`
 
-::::
+[Book website](https://bayesiancomputationbook.com/welcome.html)
+(the book website contains the whole book in online format)
```
Comment (canyon289) [inline docs/source/learning.md]:
```suggestion
(the book website contains the whole book, code examples, and notebook in online format)
```

**Review by canyon289 — APPROVED** [review]

**Comment by reshamas** [comment]
## Prior Menus

- [Installation](https://docs.pymc.io/en/latest/installation.html)
- [Learning](https://docs.pymc.io/en/latest/learning.html)
- [Examples](https://docs.pymc.io/projects/examples/en/latest/)
- [API Reference](https://docs.pymc.io/en/latest/api.html)
- [Community](https://docs.pymc.io/en/latest/community.html)
- [Contributing](https://docs.pymc.io/en/latest/contributing/index.html)
- [About PyMC](https://docs.pymc.io/en/latest/about/index.html)

## Proposed Menu

- [Learning](https://pymc--5684.org.readthedocs.build/en/5684/learning.html)
- [API](https://pymc--5684.org.readthedocs.build/en/5684/api.html)
- [Community](https://pymc--5684.org.readthedocs.build/en/5684/community.html)
- [Contributing](https://pymc--5684.org.readthedocs.build/en/5684/contributing/index.html)
- [About Us](https://pymc--5684.org.readthedocs.build/en/5684/about/index.html)
- [Examples](https://docs.pymc.io/projects/examples/en/latest/)

--- 

Reference:  https://github.com/pymc-devs/pymc/issues/5675

SUGGESTIONS:
- "About Us" should be first, what do you think?  Also, think it would be better to leave it as "About PyMC" or event "About" instead of "About Us" (because "About Us" sounds like it is about people)
- **"Consulting" menu item needed to point to: https://www.pymc-labs.io/**

File: docs/source/learning.md [hunk 3]
```diff
@@ -52,29 +65,52 @@ PyMC's own {doc}`glossary` defines many core terms and provides useful reference
...
+Probability written by members of the PyMC team.
+For the complete list see the {ref}`books` page.
 
-TODO
-:::
+#### Bayesian Modeling and Computation in Python
+
+By Osvaldo Martin, Ravin Kumar and Junpeng Lao
+
+`<add description here>`
```
Comment (OriolAbril) [inline docs/source/learning.md]: @canyon289 @aloctavodia @junpenglao do you want to add a paragraph here similar to https://docs.pymc.io/en/latest/learning.html#bayesian-methods-for-hackers and https://docs.pymc.io/en/latest/learning.html#bayesian-analysis-with-python ones?

File: docs/source/learning.md [hunk 4]
```diff
@@ -52,29 +65,52 @@ PyMC's own {doc}`glossary` defines many core terms and provides useful reference
...
-:::{grid-item-card} TODO
+### {octicon}`repo;1em;sd-text-info` Intermediate books
+There are several great books available to deepen your understanding
+of Bayesian statistics. Many of them also have their code
+examples translated to PyMC and publicly available at
+the [pymc-resources](https://github.com/pymc-devs/pymc-resources) repository.
 
+In this section we highlight one book that uses PyMC and TensorFlow
+Probability written by members of the PyMC team.
+For the complete list see the {ref}`books` page.
```
Comment (OriolAbril) [inline docs/source/learning.md:86]: not sure who to tag or if I should open an issue instead. This links to the books page, this one: https://github.com/pymc-devs/pymc/blob/main/docs/source/learn/books.md. It should have a list of books in pymc-resources and using pymc themselves as well as links to their websites and pymc code examples (can be added in a follow up or parallel PR)

File: docs/source/index.md [hunk 5]
```diff
@@ -153,11 +153,9 @@ See [Google Scholar](https://scholar.google.de/scholar?oi=bibs&hl=en&authuser=1&
...
 
-installation
 learning
-Examples <https://docs.pymc.io/projects/examples/en/latest/>
-api
+API <api>
 community
 contributing/index
-about/index
+About <about/index>
```
Comment (OriolAbril) [inline docs/source/index.md]: I have modified this to "About" only, but with "About PyMC" as the title in the page. It is still unclear what will happen with https://github.com/pymc-devs/pymc.io which might end up hosting the about page with the testimonials, blog posts and history. The theme only allows that external links have the []/ icon if they are at the end of the navbar. Keeping it here for now would mean minimal change if eventually we "outsource" the about page to pymc.io.

**Review by OriolAbril — COMMENTED** [review]
There are still many TODOs in other pages, but I don't plan to venture out of the pages already edited unless necessary. The running the test suite page is only edited to move the content from CONTRIBUTING to there, but it is still a todo page for future PRs. The homepage probably needs some work too.

File: docs/source/learning.md (hunk 3 above)
Comment (canyon289) [inline docs/source/learning.md]:
```suggestion
Bayesian Modeling and Computation in Python aims to help beginner Bayesian practitioners to become intermediate modelers. It uses a hands on approach with PyMC and ArviZ focusing on the practice of applied statistics with references to the underlying mathematical theory.
```

File: docs/source/index.md (hunk 5 above)
Comment (OriolAbril) [reply docs/source/index.md]: Related to https://github.com/pymc-devs/pymc.io/pull/4

File: docs/source/learning.md (hunk 4 above)
Comment (OriolAbril) [reply docs/source/learning.md:86]: I found a stale branch where @martinacantaro had already done this. I copied it over.

File: docs/source/index.md (hunk 5 above)
Comment (OriolAbril) [reply docs/source/index.md]: This is now an external link

**Comment by OriolAbril** [comment]
I think this is ready to merge now, we can continue iterating on this. Some highlights

* The "about" link now points to pymc.io. All social links are there on the sidebar. For now I have kept only 4 icons here. We can add more but I'd keep them to things we use very often. IMO, the main message should be "to interact with the pymc community and team join discourse", having too many icons there can distract from that. office hours, meetup... should all always and also be announced or cross-posted on Discourse
* The "cite pymc" was the only section in about that was pymc library specific and not pymc project wide, so I moved it to the homepage
* I have added a "Blog" link too that points directly to the blog section. I haven't yet added the consulting one as I think we should discuss #5675 first, update the governance and then add the link to the docs.
* I added a "Donate to PyMC" link to all the pages in the right sidebar, but the numfocus donate link is still pymc3. It will need to be updated at some point or it will stop working cc @canyon289 @twiecki 
* The bulk of the work was on the learning section. I have filled all the sections of the learning page with links to core functionality notebooks, to books, videos... I also renamed the "examples" folder that had the 5 notebooks that are versioned and automatically reexecuted when building the documentation to "core_notebooks" to keep the examples word "dedicated" to pymc-examples.

**Comment by reshamas** [comment]
@OriolAbril 
Can we see a preview of this pull request, such as on Netlify?

**Comment by OriolAbril** [comment]
@reshamas all pymc repos have previews automatically generated for their docs. You have to search for the readthedocs CI check and click on details once passed. In this one I have also copied the preview link at the PR description (rtd generated all previews of a PR at the same link,  overwriting the previews of old commits, but not of other PRs)

**Comment by reshamas** [comment]
Thanks @OriolAbril.
Can we add the Meetup link too? Or should I open up a new issue?

**Comment by OriolAbril** [comment]
It is already on the sidebar of https://www.pymc.io/. I wouldn't add it yet to the docs. As I said above I think it is important that discourse has a prominent spot so that if people are unsure about where to subscribe they go to Discourse. I think it is possible with our theme to configure the colors of the icons, one option could be having all icons be grey except the discourse one, but I can't look into it right now.

**Pull request merged**